entrypoint.sh

```diff
@@ -19,9 +20,17 @@ fi
 echo "[action-pylint] pylint version:"
 pylint --version
 
+if [[ "$INPUT_PYLINT_RC" == "" ]]; then
+  # Do not supply the rcfile option if it is not provided
+  rcfile_option=""
+else
+  rcfile_option="--rcfile=\"${INPUT_PYLINT_RC}\""
+fi
+
 echo "[action-pylint] Checking python code with the pylint linter and reviewdog..."
 exit_val="0"
-pylint --rcfile "${INPUT_PYLINT_RC}" -s n . 2>&1 | # Removes ansi codes see https://github.com/reviewdog/errorformat/issues/51
+
+pylint --score n ${rcfile_option} ${INPUT_PYLINT_ARGS} ${INPUT_GLOB_PATTERN} 2>&1 | # Removes ansi codes see https://github.com/reviewdog/errorformat/issues/51
```

Comment:
Please check this warning.

```
Double quote to prevent globbing and word splitting. [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086)
```

Reply:
```suggestion
pylint --score n "${rcfile_option}" "${INPUT_PYLINT_ARGS}" "${INPUT_GLOB_PATTERN}" 2>&1 | # Removes ansi codes see https://github.com/reviewdog/errorformat/issues/51
```

Reply:
When INPUT_GLOB_PATTERN is "**/*.py" for example, I tried `"${INPUT_GLOB_PATTERN}"` and it seems to not behave as expected.

Reply:
I can test it again